ROLES.md

```diff
@@ -164,6 +165,47 @@ a 180 day period, than that individual may lose membership. On-going contributio
 
 Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
+## Documentation Reviewer
+
+A **Documentation Reviewer**, or just **Reviewer**, is trusted to only approve
```

Comment:
is this only for documentation reviewer or general reviewer?

We purposefully got rid of general reviewer... I'm okay to introduce doc reviewer if @frankbu agrees to.

Reply:
Yes, @linsun this role is only for reviewing documentation.

Reply:
Done, thank you @linsun